tests/cc13b/input.dat

```diff
@@ -0,0 +1,50 @@
+#! Tests RHF CCSD(T)gradients
+memory 500 mb
+
+molecule h2o {
+0 1
+O
+H 1 R
+H 1 R 2 A
+
+R = 1.0
+A = 104.5
+}
+
+set {
+   reference rhf
+   basis cc-pvdz
+   r_convergence 13
+   e_convergence 13
+   d_convergence 13
+}
+
+analytic, wfn = gradient('ccsd(t)', return_wfn=True)
+
+set findif points 5
+fin_diff, wfn = gradient('ccsd(t)', dertype=0, return_wfn=True)
+
+compare_matrices(analytic, fin_diff, 10, "RHF-CCSD(T) gradient test") #TEST
```

Comment:
@loriab Can I hard code the cfour gradient results in my test case or do you want me to run a wrapper like: energy('cfour') and then compare my results?